docs/reference/ml/df-analytics/apis/infer-trained-model-deployment.asciidoc

```diff
@@ -0,0 +1,116 @@
+[role="xpack"]
+[testenv="basic"]
+[[infer-trained-model-deployment]]
+= Infer trained model deployment API
+[subs="attributes"]
+++++
+<titleabbrev>Infer trained model deployment</titleabbrev>
+++++
+
+Evaluates a trained model.
+
+[[infer-trained-model-deployment-request]]
+== {api-request-title}
+
+`POST _ml/trained_models/<model_id>/deployment/_infer`
+
+////
+[[infer-trained-model-deployment-prereq]]
+== {api-prereq-title}
+
+////
+////
+[[infer-trained-model-deployment-desc]]
+== {api-description-title}
+
+////
+
+[[infer-trained-model-deployment-path-params]]
+== {api-path-parms-title}
+
+`<model_id>`::
+(Optional, string)
```

Comment:
Thanks!  I've changed to "required" in 024b0495864b7d31a71d579a9945aeb13c780603. I haven't mentioned _all or * since we generally callout when they *are* supported.